src/index.tsx

```diff
@@ -177,34 +185,44 @@ export default class OtpInputs extends PureComponent<Props, State> {
       selectTextOnFocus,
       testIDPrefix,
       unfocusedBorderColor,
+      forceLTR,
+      isRTL,
+      debug,
```

Comment:
I would like to not keep some `debug` props as basicly it won't have any usage instead of development of this lib. Please remove it.